code/game/turfs/turf.dm

```diff
@@ -105,7 +105,8 @@
 					M.inertia_dir = 0
 			if(TURF_WET_LUBE)
 				M.slip(0, 4, null, (SLIDE|GALOSHES_DONT_HELP))
-				M.confused += 8
+				if(M.slip(0, 4, null, (SLIDE|GALOSHES_DONT_HELP))) 
```

Comment:
Since you have to change this again anyway please just remove the confusion since it ruins the puzzle @Cheridan made